org.testeditor.web.backend.persistence/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM openjdk:8-jdk
+
+# This image needs one environment variables upon execution:
+# * TARGET_REPO
+#   Defines the git repository used (read only)
+
+LABEL license="EPL 1.0" \
+      name="testeditor/persistence"
+
+ENV PROG_DIR=/opt/testeditor \
+    GRADLE_USER_HOME=/opt/testeditor/.gradle \
+    REPO_ROOT=/repo \
+    _JAVA_OPTIONS="-Djdk.http.auth.tunneling.disabledSchemes=" \
```

Comment:
I think use of this environment variable is deprecated in favor of `JAVA_TOOL_OPTIONS`; see
https://stackoverflow.com/questions/17781405/information-about-java-options#answer-40736904, https://bugs.openjdk.java.net/browse/JDK-4971166